java/README.md

```diff
@@ -11,14 +11,21 @@ This design ensures that updates easily propagate to each language and reduces o
 
 ## System Requirements
 
-The beta release of GLIDE for Valkey was tested on Intel x86_64 using Ubuntu 22.04.1, Amazon Linux 2023 (AL2023), and macOS 12.7 (aarch64-apple-darwin).
+The release of GLIDE for Valkey was tested on the following platforms:
+Linux:
+- Ubuntu 22.04.1 (x86_64)
+- Amazon Linux 2023 (AL2023) (x86_64)
+
+macOS:
+- macOS 12.7 (Apple silicon)
+
```

Comment:
I think you can add mac with intel CPU (x86_64) - @jonathanl-bq has one. This also runs on CI.